manifests/cf-manifest/isolation-segments/prod/govuk-notify-production.yml

```diff
@@ -0,0 +1,4 @@
+---
+number_of_cells: 24
```

Comment:
This is the same number of cells in our staging segment. Load tests have been good on that. One thing I've considered, but think we might be OK without is to add a few extra cells in production, just in case. Then after a few days of seeing that things are fine we could bring it down to 24 if appropriate.

Any metrics on your side to help support whether 24 is a good enough number would be useful but at the moment I feel reasonably confident.